src/utils/helper-functions/utils.ts

```diff
@@ -188,6 +189,19 @@ export function getImportRemoveFix(
   ])
 }
 
+export function getNodeToCommaRemoveFix(
```

Comment:
Not sure if it's a good name 😄

Reply:
what about removeNodeAndTrailingComma ?

Reply:
I think we can move on with this name, as it would be probably replaced by `ASTUtils. getNodeToCommaRemoveFix` (yes, for lack of a better name, I used that exact name there 😆) from `@angular-eslint/utils` in a short period. WDYT?

Reply:
👍👍